docs/user_guide/sd_fingertips.rst

```diff
@@ -1,6 +1,99 @@
 Fingertips
 ============
 
+STF Sensor
+----------
+The Shadow Tactile Fingertip (STF) sensors are magnetic based tactile sensors that are fitted
+in the standard fingertips or (slighlty bigger) thumb shapes. 
+
+.. figure:: ../img/sd_stf_sensors.png
+    :width: 40%
+
+It leverages 17 magnetic sensors evenly distributed at its core. For each sensor, a magnet is
+placed on top of a layer of silicone (flesh) that is then recovered by another layer of
```

Comment:
I would suggest "covered" instead of "recovered"

Reply:
Thanks! Addressed in https://github.com/shadow-robot/sr_dexterous_hand_documentation/pull/179/commits/66bb6dfef57d154b72092a475a02c1296d76c375